### ISSUE TITLE
Export dialog parameter display on Windoze

### ISSUE THREAD

**Issue by feathered-arch** [colin0brass/FaceLabellingExportLrPlugin#6]
https://github.com/feathered-arch/FaceLabellingExportLrPlugin/blob/master/FaceLabellingExport.lrplugin/FLEExportDialogs.lua

I have no idea what I'm doing on Github. Until this project I didn't even know release tags were a thing. :)
So I will spare you the pain of a pull request, and just put this here for you to examine and copy as you please.

Basically, numbers like 20.0 and 5.0 in the parameter boxes weren't fully visible because the width doesn't account for the decimal point itself. This may be a Windoze quirk.

**Comment by colin0brass** [comment]
Thanks @feathered-arch.

I suspect this is a quirk of the fact that I have chosen to give both slider control and edit_field, since I feel that gives good visual indication of the range supported, whilst allowing specific value entry.

The Lightroom API gives good control of the precision for edit_field, but not for slider, hence it can result in rather more decimal places than I think is relevant.

I have thought of a way to truncate the decimal places myself, but it requires adding a callback on each value change.

Before I go through the complexities of implementation, please can you help confirm which specific parameter boxes you saw this issue on and what sort of value you think it was trying to display?

Regards
Colin

**Comment by feathered-arch** [comment]
Yes, you're correct, it's the sliders. All the sliders, except for the font
size which is rounded to the nearest integer, and needs no change.

It's clear what it's trying to display - it's simply that it's visually
truncated. You can enter the field and use the left right keys to see the
entire value. You just can't see it all at once if the box isn't large
enough.

Let me know if that answers your question, or I can send a screenshot
tomorrow.

**Comment by colin0brass** [comment]
I have released v1.5 with improved slider value display. It should now round (via callback on each value change) all the non-integer slider values to 1 decimal place.
@feathered-arch , please could you help test and confirm if this resolves the issue for you? Please can you then either close the issue if resolved, or add further notes otherwise?

**Comment by feathered-arch** [comment]
Issue 6
![Issue 6](https://user-images.githubusercontent.com/37090381/111034999-2c119a80-83e6-11eb-89ac-d2bb2a44b4ef.PNG)

Issue 6_1
![Issue6_1](https://user-images.githubusercontent.com/37090381/111035000-2caa3100-83e6-11eb-8056-700e13048d53.PNG)

Issue 6_2
![issue6_2](https://user-images.githubusercontent.com/37090381/111035001-2caa3100-83e6-11eb-83c1-5b9c668d7e9e.PNG)

Issue 6_width_in_digits
![width_in_digits](https://user-images.githubusercontent.com/37090381/111035002-2caa3100-83e6-11eb-82b7-561502228d7b.PNG)

Issue 6_fixed
![issue6_width_in_digits_fixed](https://user-images.githubusercontent.com/37090381/111035003-2caa3100-83e6-11eb-93a8-722ed241718d.PNG)

So, apologies for not being clear. All I did to fix the issue was change the width_in_digits value. 

In the Issue6_2 image you see the appearance with the existing code; however, since it's only 3 characters ("0.5") there is no issue. But 4 characters ("20.0") does not fit.

**Comment by colin0brass** [comment]
Ah, thanks for the screenshots. I see that Windows was showing decimal places for label options and line width fields that I intended to be integers.

On closer inspection of the code, I found that I was being inconsistent in my 'precision' vs 'increment' settings for the edit_fields, and it looks like Lightroom was interpreting that differently on Windows vs Mac, hence I hadn't seen it myself.

I have a draft update with those corrected, and have also increased the width_in_digits on some of the other fields, as per your suggestions.

I am also looking at a different issue, and will aim to fix both in the next release.
